detections/application/m365_copilot_agentic_jailbreak_attack.yml

```diff
@@ -0,0 +1,56 @@
+name: M365 Copilot Agentic Jailbreak Attack
+id: e5c7b380-19da-42e9-9e53-0af4cd27aee3
+version: 1
+date: '2025-09-25'
+author: Rod Soto
+status: experimental
+type: TTP
+description: Detects agentic AI jailbreak attempts that try to establish persistent control over M365 Copilot through rule injection, universal triggers, response automation, system overrides, and persona establishment techniques.
+search: >
+  `m365_exported_ediscovery_prompt_logs`
```

Comment:
Can you perhaps create a data source file for this as well and instructions on how to export and ingest this csv based logs? 

Also, improve the same on how to implement for all `m365_exported_ediscovery_prompt_logs`

Reply:
Done